bookkeeper-server/src/test/java/org/apache/bookkeeper/server/http/TestHttpService.java

```diff
@@ -838,12 +858,14 @@ public void testGCDetailsService() throws Exception {
         HttpServiceRequest request1 = new HttpServiceRequest(null, HttpServer.Method.GET, null);
         HttpServiceResponse response1 = gcDetailsService.handle(request1);
         assertEquals(HttpServer.StatusCode.OK.getValue(), response1.getStatusCode());
+        assertTrue(JsonUtil.fromJson(response1.getBody(), List.class).get(0).toString().contains("ledgerDir"));
```

Comment:
Please check all the contents of your output instead of testing whether the `ledgerDir` exists or not.